src/components/TextBox/index.tsx

```diff
@@ -560,8 +560,8 @@ const TextBox = forwardRef<
         <StyledRelativeDiv>
           <StyledInput
             aria-controls={ariaControls}
-            aria-label={label && noTopLabel ? label : undefined}
-            aria-labelledby={ariaControls}
+            aria-label={label && noTopLabel ? label : name}
```

Comment:
An input name is not a label, a11y-wise this is kinda useless :)

Reply:
@chambo-e all good now